### ISSUE TITLE
Catch the exception while twitter service is under maintaince.

### ISSUE THREAD

**Issue by sishen** [mbleigh/twitter-auth#12]
Sometimes twitter is under maintaince and as a result the app can't request the oauth token. We need to do something instead of directly raising the exception to the top, such as catch the exception in the SessionsController#new,

  rescue Exception => e
    authentication_failed("We're currently experiencing issues with Twitter's OAuth Service. Please try later.") and return

**Comment by mbleigh** [comment]
Yeah, I noticed that as well. Will work in a catch for that sometime soon.
